package.json

```diff
@@ -3,6 +3,18 @@
   "version": "0.0.0-development",
   "description": "OpenApi TypeScript client generator",
   "main": "lib/index.js",
+  "exports": {
```

Comment:
Note that this restricts all previous exports as well, so the following will no longer work:

```ts
import * as Oazapfts from "oazapfts/lib/runtime/index.js";
import * as QS from "oazapfts/lib/runtime/query.js";
```

So this might be considered a breaking change, unless all previously existing paths are added to the `export` config as well.

Reply:
Good point!